packages/common/src/ui/layout/tables/RemoteDataTable.tsx

```diff
@@ -145,19 +164,18 @@ export const RemoteDataTable = <T extends Record<string, unknown>>({
           {rows.map((row: Row<T>) => {
             prepareRow(row);
 
-            const { cells, values } = row;
+            const { cells, original } = row;
             const { key } = row.getRowProps();
 
             return (
               <DataRow<T>
                 cells={cells}
-                values={values as T}
                 key={key}
                 onClick={onRowClick}
+                rowData={original}
```

Comment:
oops I got confused. `values` is the values actually in the table, which doesn't include `color` or `id`